tests/Homework_6_VG.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+# from time import sleep
+from time import sleep
+
+from selenium import webdriver
+from selenium.webdriver import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support.wait import WebDriverWait
+
+from PythonBootcamp_Summer2021.tests import DOMAIN, CHROME_PATH, ADMIN_USER, DEFAULT_PASSWORD
+
+
+class LoginPageTest(unittest.TestCase):
+    def setUp(self) -> None:
+        self.driver = webdriver.Chrome(executable_path=CHROME_PATH)
+        self.driver.get(DOMAIN)
+
+
+    def test_valid_logging(self, expected_condition=None):
+        original_url = self.driver.current_url
+        self.driver.find_element(By.ID, "txtUsername").send_keys(ADMIN_USER)
+        self.driver.find_element(By.ID, "txtPassword").send_keys(DEFAULT_PASSWORD)
+        self.driver.find_element(By.ID, "btnLogin").click()
+
+        wait = WebDriverWait(self.driver, 3)
+        wait.until(expected_conditions.url_contains("/pim/viewEmployeeList"))
+
+        new_url = self.driver.current_url
+        self.assertNotEqual(original_url, new_url)
+        self.assertEqual("Employee Information", self.driver.find_element(By.TAG_NAME, "h1").text)
+
+    def test_filter_results_by_employee_id(self):
+        self.driver.find_element(By.ID, "txtUsername").send_keys(ADMIN_USER)
+        self.driver.find_element(By.ID, "txtPassword").send_keys(DEFAULT_PASSWORD)
+        self.driver.find_element(By.ID, "btnLogin").click()
+        self.driver.find_element(By.ID,"empsearch_id").send_keys("0001")
+        self.driver.find_element(By.ID, "searchBtn").click()
+        self.assertEqual("0001", self.driver.find_element(By.LINK_TEXT, "0001").text)
```

Comment:
These 3 asserts are problematic, since they would either result in NoSuchElementException or pass... the AssertError would never get triggered.
Because if you are looking for an element by text... AND you find it. There is no point in checking that that same element has that same text.